### ISSUE TITLE
LLM Summary MVP

### ISSUE THREAD

**Issue by sbddesign** [atlantabitdevs/bitdevs-upgrade#3]
- Run script from `yarn`
- Script takes an event markdown file and scrapes content from links in the file
- Nice to have: way for organizer to say "no I dont want this link scraped"
- Script sends scraped content to the LLM
- LLM sends back Cliff Notes and ELI5
- Script creates an addendum markdown file containing the summaries

**Comment by bnonni** [comment]
L0 Estimate: Wednesday, July 19

**Comment by sbddesign** [comment]
This is basically done. Thanks @niteshbalusu11!
